go.mod

```diff
@@ -296,3 +296,5 @@ replace github.com/benbjohnson/clock v1.3.0 => github.com/benbjohnson/clock v1.1
 
 // copy from TiDB
 replace go.opencensus.io => go.opencensus.io v0.23.1-0.20220331163232-052120675fac
+
+replace github.com/segmentio/kafka-go v0.4.38 => github.com/3AceShowHand/kafka-go v0.0.0-20230109084533-cd584ac8c705
```

Comment:
Can we fork it to pingcap org?

Reply:
the PR  to upstream in review status,  we will use kafka-go directly later

Reply:
after https://github.com/segmentio/kafka-go/pull/1059 merged, we can remove this replacement.